### PR TITLE
Updated OSS Policy Preview

### PR DESCRIPTION
**Pull request by chesta-mittal**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/oss-config-preview/1.0.0/ControlSettings.json [hunk 1]
```diff
@@ -883,6 +883,82 @@
...
+      "ControlIds": [
+        "ADO_Environment_AuthZ_Dont_Grant_All_Pipelines_Access",
+        "ADO_Environment_AuthZ_Restrict_Broader_Group_Access"
+      ]
+    },
+    {
+      "ResourceType": "Repository",
+      "ControlIds": [
+        "ADO_Repository_AuthZ_Dont_Grant_BuildSvcAcct_Permission",
+        "ADO_Repository_AuthZ_Dont_Grant_BuildSvc_Permission_On_Branch"
```
Comment (rbavdekar) [inline src/oss-config-preview/1.0.0/ControlSettings.json:959]: pleas change prod config as well else you wil have to make another pr

Comment (chesta-mittal) [reply src/oss-config-preview/1.0.0/ControlSettings.json:959]: Made the changes

**Review by rbavdekar — APPROVED** [review]

**Review by saikumarra — APPROVED** [review]
